libqtile/resources/default_config.py

```diff
@@ -160,6 +160,7 @@
 dgroups_app_rules = []  # type: list
 follow_mouse_focus = True
 bring_front_click = False
+floats_kept_above = False
```

Comment:
should we set this to True by default maybe?

Reply:
I didn't do that because it would change current behaviour (which often gets a negative reaction) however, I don't feel particularly strongly about it as I rarely use floating windows!

Reply:
Yeah makes sense. It's really the fact that I've seen multiple people annoyed by the default behaviour. But it's an easy toggle so it doesn't matter that much

Reply:
I think we should enable it by default. This won't affect existing users because they already have configs. Those that complain are often new users who are frustrated that they can't move focus to windows hidden at the back. This way that doesn't even happen, and when they reach that point in the config (i expect many new users will start qtile with the defaults before ever even opening the config!) they'll see that it can be disabled.